src/Mapbender/CoreBundle/Controller/ApplicationController.php

```diff
@@ -393,7 +400,28 @@ public function instanceTunnelAction($slug, $instanceId)
                 $url = $source->getGetFeatureInfo()->getHttpGet();
                 break;
             case 'getlegendgraphic':
-                $url = $source->getGetLegendGraphic()->getHttpGet();
+                $glgMode = $request->query->get('_glgmode', null);
```

Comment:
Not the best naming. Hidden request option? :)

Reply:
"Hidden" because it's definitely not a standard WMS parameter.
glg = short for GetLegendGraphic.

Reply:
ok, I understand.